theano/gpuarray/tests/test_GpuMaxAndArgmax.py

```diff
@@ -0,0 +1,139 @@
+from unittest import TestCase
+from theano.tests import unittest_tools as utt
+import numpy as np
+import theano
+import theano.tensor as T
+
+
+def randomTensor(*shapes):
+    dimlist = shapes
+    size = 1
+    for dimsize in dimlist:
+        size *= dimsize
+    return np.random.normal(size=size).astype(np.float32).reshape(dimlist)
```

Comment:
There is a function called rand_gpuarray in test_basic_ops that you should use instead.